### PR TITLE
Use Emscripten SDKs in WASM workload

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Review by lewing — COMMENTED** [review]
Manifest and target changes look good to me.

**Comment by pranavkm** [comment]
I'm happy to wait for the [runtime update](https://github.com/dotnet/aspnetcore/pull/31255) to go thru before merging this. Prepping this up in case that continues to take a while.

File: eng/Version.Details.xml
Comment (dougbu) [inline eng/Version.Details.xml:270]: Recommend aligning the SHAs w/ #31255 to avoid conflicts between this and that. But, I agree it'll be better all 'round if that goes in first (with new versions from dotnet/runtime).

**Comment by pranavkm** [comment]
@dougbu since this build is passing, would you be ok if we get this in and I'll follow up in the dependency update PR to fix up things?

**Comment by dougbu** [comment]
@pranavkm this change has not been approved and we're a week passed the Code Complete date. Need to hear from @Pilchie and a reviewer&hellip;

**Comment by Pilchie** [comment]
Given that we don't have the Preview 3 build to validate yet, I'm okay with sneaking this in.

**Comment by dougbu** [comment]
Thanks @Pilchie and sorry for contacting on a day off.

@pranavkm I guess that means this was an ask-mode change and @Pilchie answered😁 I marked it up as servicing-approved. Just need an approver&hellip;

**Comment by pranavkm** [comment]
@lewing would you mind signing off? You have the most context here.

**Review by lewing — APPROVED** [review]

**Comment by dougbu** [comment]
/fyi I fixed up the conflicts in #31255

**Pull request merged**
